### ISSUE TITLE
Support for rsa-sha256

### ISSUE THREAD

**Issue by rohitmichael** [onelogin/dotnet-saml#10]
Are we expecting support for sha256 anytime soon?

**Comment by pitbulk** [comment]
Yes, the new release will support sha256. We want to release it soon.

**Comment by GregSawin** [comment]
FYI, I followed these instructions to add support for sha256: http://samlcomponent.net/working-sha-256/
